src/gencost/constants.py

```diff
@@ -141,4 +142,90 @@
     "age_of_observation",
     "age_relative_to_prime_avg",
     "pollution_control_costs_per_kw",
+    "generator_operating_date",
+]
+
+HIST_EP_COLS = [
+    "biofuel_mmbtu",
+    "coal_mmbtu",
+    "natural_gas_mmbtu",
+    # "nuclear_mmbtu",
+    "other_mmbtu",
+    "other_gas_mmbtu",
+    "petroleum_mmbtu",
+    "petroleum_coke_mmbtu",
+    # "renew_mmbtu",
+    "biofuel_net_mwh",
+    "coal_net_mwh",
+    "natural_gas_net_mwh",
+    # "nuclear_net_mwh",
+    "other_net_mwh",
+    "other_gas_net_mwh",
+    "petroleum_net_mwh",
+    "petroleum_coke_net_mwh",
+    # "renew_net_mwh",
+    "generator_starts",
+    "fuel_starts",
+    "gross_generation_mwh",
+    "biofuel_gross_mwh",
+    "coal_gross_mwh",
+    "natural_gas_gross_mwh",
+    "other_gross_mwh",
+    "other_gas_gross_mwh",
+    "petroleum_gross_mwh",
+    "petroleum_coke_gross_mwh",
+    "biofuel_fraction",
+    "coal_fraction",
+    "natural_gas_fraction",
+    "other_fraction",
+    "other_gas_fraction",
+    "petroleum_fraction",
+    "petroleum_coke_fraction",
+    "minor_fuels_fraction",
+    "net_generation_mwh",
+    "gross_cf",
+    "wage_scale",
+    "age_of_observation_secular_adj",
+    "inflator_to_2021"
```

Comment:
In future revision, consider merging these on from source at the end rather than having them come either from historical comparables or the current generator value.